deployer/file_acquisition.py

```diff
@@ -107,26 +107,19 @@ def get_decrypted_file(original_filepath):
         # if the file is valid JSON/YAML, and then if it has a `sops` key
         with open(original_filepath) as f:
 
-            # FIXME: Right now we expect encrypted files to be JSON or YAML files, and
-            #        so we fail if these are not valid JSON/YAML. However in the
-            #        future, we may want to support encrypted files of other types
-            #        and we should update this section accordingly.
-            #
             # Support the (clearly wrong) people who use .yml instead of .yaml
             if ext == ".yaml" or ext == ".yml":
```

Comment:
All JSON is valid YAML as well by design, so you can actually just use the YAML parser here to load files unconditionally, without having to rely on extensions. If a YAML parser can't read it, then it's not encrypted by sops.

Reply:
Done!